docs/admin/tutorial.md

```diff
@@ -50,29 +51,25 @@ sudo ./target/debug/bpfd&
 
 ### Step 4: Load your first program
 
-We will load the simple `xdp-pass` program, which permits all traffic to the interface eth0.
+We will load the simple `xdp-pass` program, which permits all traffic to the attached interface,
+`vethb2795c7` in this example.
```

Comment:
why did you switch from something simple like `eth0` to a veth here?

Reply:
Because I was creating actual examples and that's what I use on my system. I kept forgetting to go back and change `vethb2795c7` back to `eth0`, and on some of the lists, I kept having to adjust the spacing of the output. So gave up and just used my port in the examples.